dashboard/pages/screening/callbacks.py

```diff
@@ -574,6 +577,21 @@ def on_report_generate_button_click(
     ), dict(content=jinja_template, filename=filename)
 
 
+def on_json_generate_button_click(
+    n_clicks,
+    report_data_second_stage: dict,
+    report_data_third_stage: dict,
+    report_data_csv: dict,
+    file_storage: FileStorage,
+):
+    filename = f"program_settings_{datetime.now().strftime('%Y-%m-%d')}.json"
+    dict_to_json = generate_json_data(
```

Comment:
as this variable is still a dict and not a JSON, consider renaming it to e.g. process_settings